structuredevent-service-cdp/src/main/java/com/sequenceiq/cloudbreak/structuredevent/service/db/CDPStructuredEventDBService.java

```diff
@@ -63,7 +63,8 @@ public void create(CDPStructuredEvent structuredEvent) {
 
     private ValidationResult validate(CDPStructuredEvent event) {
         ValidationResult.ValidationResultBuilder builder = ValidationResult.builder();
-        if (StringUtils.isEmpty(event.getOperation().getResourceCrn())) {
+
+        if (!StringUtils.hasText(event.getOperation().getResourceCrn())) {
```

Comment:
Shouldn't we check if this is a valid crn or not by using `Crn.isCrn(resourceCrn)` method?

Reply:
My only intention was to swap out the deprecated `.isEmpty` method with the `.hasText` method. I can make sure the CRN is valid too, if you'd like.

Reply:
ok